src/utils/dateUtils.ts

```diff
@@ -1,20 +1,19 @@
 import { Dayjs } from 'dayjs';
 
-export const getNaiveDate = (dateString: string): Date => {
-    const date = new Date(dateString);
-    return new Date(
-        date.getUTCFullYear(),
-        date.getUTCMonth(),
-        date.getUTCDate(),
-        date.getUTCHours(),
-        date.getUTCMinutes(),
-    );
-};
-
 export const getNewDateWithOffset = (date: Date, offset: number ):Date => {
     return new Date(new Date(date).setDate(date.getDate() + offset));
 };
 
+/**
+ * Removes the offset part of an ISO datetime string.
+ *
+ * This is needed because event objects send an offset of +00:00 (UTC Time) from the server.
+ * It is not needed if the datetime string coming from the server doesn't contain an offset.
+ */
+export const removeOffset = (datetime: string): string => {
+    return datetime.split('+')[0];
+};
```

Comment:
This is a perfect candidate for unit testing. If we manage to merge #424 before this one, I think you should add a small unit test. We can collaborate! I expect @djbusstop's PR to be merged tomorrow. 😊